gtk4/src/string_object.rs

```diff
@@ -0,0 +1,59 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::StringObject;
+use glib::{GStr, GString};
+
+impl From<StringObject> for String {
```

Comment:
```suggestion
impl<'a> From<&'a StringObject> for String {
```

etc? Or maybe in addition? There's no reason to consume the object here

Reply:
I added that in addition, thanks